README.md

```diff
@@ -19,6 +19,8 @@ Used by the [Dart Package site](https://pub.dev/).
 > pub global activate pana
 ```
 
+NOTE: You may need to restart your IDE for pana commands to be recognized.
```

Comment:
Is this a general thing for pub global activate?

What ide did you experience this with?